crates/unftp-auth-jsonfile/files/run.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
+
+function error {
+    RED='\033[0;31m'
+    NO_COLOR='\033[0m'
+    echo -e ${RED}$*${NO_COLOR} >&2
+}
+
+function exit_fail {
+    error $*
+    exit 1
+}
+
+[[ $BASH_VERSION =~ ^5 ]] || exit_fail "This script needs to run with Bash version 5"
```

Comment:
Do you intend that people can run it outside the container too?

Reply:
they might, but this was more like a basic sanity check... there are lots of other assumptions in the script (such as availability of utils that are added with apk)